<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VariantMapEditor</class>
 <widget class="QWidget" name="VariantMapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QToolButton" name="addRowButton">
       <property name="toolTip">
        <string>Добавить новую строку</string>
       </property>
       <property name="statusTip">
        <string>Добавить новую строку</string>
       </property>
       <property name="whatsThis">
        <string>Добавить новую строку</string>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeRowBtn">
       <property name="toolTip">
        <string>Удалить выбранные строки</string>
       </property>
       <property name="statusTip">
        <string>Удалить выбранные строки</string>
       </property>
       <property name="whatsThis">
        <string>Удалить выбранные строки</string>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearButton">
       <property name="toolTip">
        <string>Очистить таблицу</string>
       </property>
       <property name="statusTip">
        <string>Очистить таблицу</string>
       </property>
       <property name="whatsThis">
        <string>Очистить таблицу</string>
       </property>
       <property name="text">
        <string>Очистить таблицу</string>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
